.project-template/fill_template_vars.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Is this script needed?

Reply:
It's handy, if we ever decide to change the name or the description etc, it will go through all places that needs to be updated. It's all based on our default scaffolding that we use for all projects https://github.com/ethereum/ethereum-python-project-template